.github/workflows/dockerbuild.yml

```diff
@@ -45,6 +45,8 @@ jobs:
         with:
           platforms: linux/arm64,linux/amd64,linux/arm/v6,linux/arm/v7
           file: docker/Dockerfile
+          cache-from: type=registry,ref=zwavejs/zwavejs2mqtt:dev
+          cache-to: type=inline
```

Comment:
Shouldn't this also have the  `mode=max`

Reply:
Yes. If we keep multi step builds